src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER = 100;
```

Comment:
Better to have more descriptive constant names
```suggestion
    private static final int BALL_NUMBER_LIMIT = 100;
```

Reply:
done